### PR TITLE
Add onload animations script

### PR DESCRIPTION
**Pull request by alexcostaluiz**
Add a script to handle the simple page load animations.

This script will be included in all html pages. The script registers an even handler for the DOMContentLoaded event, upon which the appropriate fade shift animations will be started according to which DOM elements are found in the html document.

### CODE REVIEW

File: portfolio/src/main/webapp/js/onload-animations.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+});
+
+/**
+ * Fades in and shifts up any elements of class `bp`.
+ */
+function fadeShiftContent() {
+  const body = document.querySelectorAll('.bp');
+  for (let i = 0; i < body.length; i++) {
+    body[i].id = 'bp' + i;
+    Animator.queue(new FadeShift(body[i], 1, 0.1, 'y', 24, 0, -1.6));
```
Comment (surbhimadan1995) [inline portfolio/src/main/webapp/js/onload-animations.js:33]: Lets add param tags to the various numbers here so its easier to read at which number represents what 
example
new FadeShift(body[i], /* heightPx= */ 1, /* ratio= */ 0.1 ...etc );   
(I don't actually know what these numbers represent so my tags are made up names but you should use the actual names in the method declaration)

same for the calls below

File: portfolio/src/main/webapp/js/onload-animations.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ * 05/29/2020
+ *
+ * @author Alexander Luiz Costa
+ */
+
+/*
+ * Starts the appropriate page load animations on DOMContentLoaded
+ * event.
+ */
+window.addEventListener('DOMContentLoaded', () => {
```
Comment (jesseelizabeth) [inline portfolio/src/main/webapp/js/onload-animations.js:12]: Should this event listener be on the window or the document?

Comment (alexcostaluiz) [reply portfolio/src/main/webapp/js/onload-animations.js:12]: Ahh you are right, it belongs on the document. Thanks!

File: portfolio/src/main/webapp/js/onload-animations.js (hunk 1 above)
Comment (alexcostaluiz) [reply portfolio/src/main/webapp/js/onload-animations.js]: Got it, thanks!

**Pull request merged**